Add a button to the Study Intro that allows the user to continue from their last added kanji.

TylerLafayette opened the pull request:
Added a button that allows a user to continue from where they left off. I think this would be a helpful addition to the site as I find it inconvenient to have to enter the correct number to get to where I left off.
**Known issue:** I think this only works with RTK1 at the moment.
<img width="714" alt="Screen Shot 2019-11-15 at 5 03 33 PM" src="https://user-images.githubusercontent.com/16327601/68985218-e630cc00-07c9-11ea-93d3-f1b87d18ed3a.png">

ShavingSeagull commented:
I would love to see this on the site. I thought I was going blind not being able to find the button to pick up from where I left off.

fabd commented:
Hi @TylerLafayette really sorry I don't remember seeing a notification about this pull request. It's really not obvious >_>  I saw it now due to ShavingSeagull's comment.

Indeed "continue where you left off" would be a good addition. The problem is that a user can add cards in non-sequential order.

Ideally I suppose the button would only show up for a user who has no "gaps" in their flashcard set.

I think I dealt with that conditional before -- [this is the function](https://github.com/fabd/kanji-koohii/blob/master/src/lib/peer/ReviewsPeer.php#L337)).

In fact it looks like the function you added is the same, but it does not filter cards for the particular user (doesn't use the `$userId` param). Though I'm guessing you meant to get the max index # in the user's cards -- filtered by RTK1 range -- and let user proceed from that index #49 

I suppose that conditional works for most purposes.

I appreciate the contribution, if you can fix the `$userId` issue I think I can merge that, though I'll have to spend some more time on it later. I'll also probably want to refactor `getHeisigProgressCount()` with an option to `$ignoreGaps` as both functions are very similar.

TylerLafayette commented:
Thank you for your reply! Looking back, I'm not actually sure why I thought it was necessary to make a new function. I must have assumed the preexisting one wouldn't work. Anyways, in my latest commit, I removed the function I made and changed `$this->latestKanji` to use `ReviewsPeer::getHeisigProgressCount` instead of my custom function. Thanks again for considering this feature, I'm happy to be contributing to a site I use daily.
